### ISSUE TITLE
fix attribute completeness description text

### ISSUE THREAD

**Issue by JanReifenrath** [GIScience/ohsome-quality-api#862]
The attribute completeness description text says "attributes" even if there is only a single attribute selected. It should only say "attribute"

**Comment by JanReifenrath** [comment]
the code should look roughly like this
if len(self.attribute_names) > 1:
            tags = "attributes " + ", ".join(self.attribute_names)
 else:
            tags = "attribute " + self.attribute_names[0]

**Comment by Gigaszi** [comment]
hey @JanReifenrath , didnt you already fix this or do i remember wrong?

**Comment by JanReifenrath** [comment]
yes, this has already been fixed (link to commit where it got fixed: https://github.com/GIScience/ohsome-quality-api/commit/3bfb2d5780dd29e9fe310a5bbe8be431e158f23f )
